resources/install/example.sql

```diff
@@ -19,3 +19,14 @@ CREATE TABLE IF NOT EXISTS `activities_activities` (
     KEY `row` (`row`),
     KEY `ip` (`ip`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE IF NOT EXISTS `activities_resources` (
```

Comment:
You can make the primary a UUID or, you can have a normal auto increment and add a UUID, this is required to be able to unique identify the entity, if the entity already has a UUID we should use that and not generate our own.